application/controllers/admin/DataEntry.php

```diff
@@ -620,14 +620,14 @@ public function editdata($subaction, $id, $surveyid)
                 $aDataentryoutput .= stripJavaScript($question);
                 $aDataentryoutput .= "</td>\n";
                 // Second column (Unseen)
-                $aDataentryoutput .= "<td style='width: 120px;'>\n";
+                $aDataentryoutput .= "<td style='width: 8em;'>\n";
                 // Only show the "Unseen" checkbox for questions.
                 if (!empty($fname['title'])) {
                     $aDataentryoutput .= "<div class='checkbox'>\t"
-                        . "<input type='checkbox' name='{$fname['fieldname']}_unseen' id='{$fname['fieldname']}_unseen'"
+                        . "<input type='checkbox' name='unseen_{$fname['fieldname']}' id='unseen_{$fname['fieldname']}'"
```

Comment:
But why not {$fname['fieldname']}:unseen ?

Reply:
I believe the `:` can be error prone or problematic.
The current naming schema works. So, don't want to add an extra layer of "risk" (as a way to say it) :)